mozci/console/commands/push.py

```diff
@@ -822,13 +823,15 @@ def handle(self) -> None:
                         push, "group"
                     )
 
-                if not ever_classified_as_cause:
-                    warnings.append(
-                        {
-                            "message": f"Push {push.branch}/{push.rev} was backedout and all of its failures and the ones of its children were marked as intermittent or marked as caused by another push.",
-                            "type": "error",
-                        }
-                    )
+                    if config.get("warnings", {}).get(
```

Comment:
You're not using `warnings_to_notify` here like for the other warnings

Reply:
Fixed by dropping `warnings_to_notify` in ed9738f.